dataflow/manual/examples/BusyExprSimple.java

```diff
@@ -0,0 +1,13 @@
+class Test {
+    public void test() {
+        int a = 2, b = 3;
+        if (a != b) {
+            int x = b-a;
```

Comment:
Nit: spacing around binary operators.

Reply:
content.tex autoref this java file, adding spaces will make the code wilder in manual, and on the right is the pdf file. I tried, it doesn't look good.

Reply:
Change to 2-space indentation